README.md

```diff
@@ -1 +1,14 @@
-# atom-ember-colorizer
+# ember-frost-syntax theme
+
+A short description of your theme.
+
+## Usage
+Change syntax theme to Atom Ember Frost
+
+## Theme:
+* Handlebars:
+	* ![Handlebars](images/handlebars-1.png)
+	* ![Handlebars](images/handlebars-2.png)
+
+## Notes:
+Be sure to download a Babel package that handles grammar for a good Ember-Atom experience
```

Comment:
Yeah, this was more a comment about clarifying that the grammar was to handle the ES6 format javascript to pair with the hbs grammar